crates/rspack/tests/tree-shaking/ts-target-es5/expected/main.js

```diff
@@ -45,76 +45,19 @@ function _async_to_generator(fn) {
     };
 }
 },
-"../../../../../node_modules/@swc/helpers/esm/_instanceof.js": function (module, exports, __webpack_require__) {
-"use strict";
-Object.defineProperty(exports, "__esModule", {
-    value: true
-});
-function _export(target, all) {
-    for(var name in all)Object.defineProperty(target, name, {
-        enumerable: true,
-        get: all[name]
-    });
-}
-_export(exports, {
-    _instanceof: function() {
-        return _instanceof1;
-    },
-    _: function() {
-        return _instanceof1;
-    }
-});
-var _instanceof = __webpack_require__("../../../../../node_modules/@swc/helpers/esm/_instanceof.js");
-function _instanceof1(left, right) {
-    if (right != null && typeof Symbol !== "undefined" && right[Symbol.hasInstance]) return !!right[Symbol.hasInstance](left);
-    else return _instanceof._(left, right);
-}
-},
 "../../../../../node_modules/@swc/helpers/esm/_ts_generator.js": function (module, exports, __webpack_require__) {
 "use strict";
 Object.defineProperty(exports, "__esModule", {
     value: true
 });
-function _export(target, all) {
-    for(var name in all)Object.defineProperty(target, name, {
-        enumerable: true,
-        get: all[name]
-    });
-}
-_export(exports, {
-    _: function() {
-        return _tslibes6.__generator;
-    },
-    _ts_generator: function() {
+Object.defineProperty(exports, "_", {
```

Comment:
this modification is expected, since `_ts_generator` is not used.

Reply:
![image](https://user-images.githubusercontent.com/17974631/233819884-3a61226e-2322-445b-b271-ba5c158286e7.png)
![image](https://user-images.githubusercontent.com/17974631/233819891-7b02a578-3b06-4afe-8a01-0411825fdb59.png)

only `_` is used.